pkg/osutil/dns_others.go

```diff
@@ -0,0 +1,8 @@
+// +build !darwin
```

Comment:
Could you gofmt with Go 1.17

Reply:
> Could you gofmt with Go 1.17

Sure. That also updated `pkg/osutil/osutil_others.go`